src/signals/incident/containers/KtoContainer/components/KtoForm/KtoForm.tsx

```diff
@@ -181,6 +182,7 @@ const KtoForm = ({
                     trigger: trigger,
                     setValue: setValue,
                     register: register,
+                    unregister: unregister,
```

Comment:
can be just unregister?!. Would do the same for the other methods

Reply:
Thanks!